packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -116,6 +116,11 @@ export const CdpAutomation = (sendDebuggerCommandFn: SendDebuggerCommand) => {
       }
     }
 
+    if (setCookieRequest.name.startsWith('__Host-')) {
+      setCookieRequest.url = `https://${cookie.domain}`
+      delete setCookieRequest.domain
+    }
```

Comment:
so now back to this... this is no longer necessary, since the `secure` attribute will be juggled up, correct?

```suggestion
```

Reply:
No. the only way to set a __Host- prefix cookie in chrome devtools protocol is by setting the `url` property and omitting `domain`. So this translation needs to happen for chrome. Firefox does not have this issue

Reply:
i'll be damned, i just tried it and you're right. that's so weird. ok cool beans